### ISSUE TITLE
Maintainer lookup by email, not name

### ISSUE THREAD

**Issue by kaniini** [alpinelinux/aports-turbo#38]
It is possible that two maintainers might have the same name if it is a common one.  So we should use email addresses instead.

**Comment by clandmeter** [comment]
I have been reluctant to add email address to pkgs.a.o because spam related risks. This is also the reason that package listing only shows the name and does not include the email address.

Is it worth taking the risk or do we have some way to work around it?

**Comment by kaniini** [comment]
I don't think it really matters.  A spam bot can get the emails by crawling our git repositories since cgit is available.

**Comment by kaniini** [comment]
Not to mention just pulling them out of the APKINDEX.  There is precedent for this; at least one spammer has pulled emails out of the Debian package indexes.
